components/TodoList.tsx

```diff
@@ -0,0 +1,44 @@
+import Todo from "@/components/Todo";
+import { InstaQLEntity } from "@instantdb/react-native";
+import { AppSchema } from "@/instant.schema";
+import DraggableFlatList, {
+  RenderItemParams,
+} from "react-native-draggable-flatlist";
+import { GestureHandlerRootView } from "react-native-gesture-handler";
+import { db } from "@/utilities/database";
+
+type TodoType = InstaQLEntity<AppSchema, "todos">;
+
+export default function TodoList({ todos }: { todos: TodoType[] }) {
+  const handleDragEnd = ({ data }: { data: TodoType[] }) => {
+    const updates = data.map((todo, index) => {
+      return db.tx.todos[todo.id].update({ sortOrder: index * 1000 });
```

Comment:
One problem with this is that for every change in sort order, you will be affecting all todos. This can get problematic when you have lots and lots of todos. 

To solve this you can do something called fractional indexing. For now no need, but wanted to call it out